eclass/cmake.eclass

```diff
@@ -412,6 +409,7 @@ cmake_src_prepare() {
 			chmod -R a-w "${S}"
 		fi
 		_CMAKE_PREPARE_HAS_RUN=1
+		eqawarn "QA Notice: EAPI=7 ebuild will throw unsupported EAPI error after 2025-11-01."
```

Comment:
OK. Once ::gentoo is free of them, consider upgrading to a global-scope warning for the benefit of overlays, like in 47d7fb1a65af001085f973704dfeed320111ab6b.